ghost/job-manager/lib/job-manager.js

```diff
@@ -39,7 +39,7 @@ class JobManager {
      * @param {Object} [options.domainEvents] - domain events emitter
      */
     constructor({errorHandler, workerMessageHandler, JobModel, domainEvents}) {
-        this.queue = fastq(this, worker, 1);
+        this.queue = fastq(this, worker, 3);
```

Comment:
Will defer this to @naz as he has winder context on introducing such change in job manager :)

Reply:
@naz Reverting this for now until you are back. We need to talk about this later, as I think we need to increase this a bit. When you are importing a lot of members all the jobs stall because of the concurrency of 1.
```suggestion
        this.queue = fastq(this, worker, 1);
```